src/tir/transforms/unroll_loop.cc

```diff
@@ -57,6 +58,9 @@ struct UnrollLoopConfigNode : public tvm::AttrsNode<UnrollLoopConfigNode> {
     TVM_ATTR_FIELD(explicit_unroll)
         .describe("Whether to explicitly unroll the loop instead of setting a pragma")
         .set_default(true);
+    TVM_ATTR_FIELD(explicit_unroll_max_extent)
+        .describe("The maximum extent of a loop that can be unrolled explicitly (-1 for infinite)")
+        .set_default(32);
```

Comment:
It's good to limit unrolling extent. However, would it be better to set default value as -1 so that user can still do arbitrarily unrolling by default and only limit it when necessary? Setting another default value can still potentially cause performance issue for existing tuned schedules.